packages/pf-codemods/README.md

```diff
@@ -1552,6 +1552,35 @@ Out:
 <Spinner  />
 ```
 
+### table-update-deprecatedPath [(#8892)](https://github.com/patternfly/patternfly-react/pull/8892)
+
+We've deprecated the current implementation of Table. In order to continue using this deprecated implementation, the import path must be updated to our deprecated package and specifiers must be aliased. However, we suggest updating to our composable Table implementation.
+
+The following imports will be affected by this deprecation:
+
+- Table
+- TableBody
+- TableHeader
+- TableProps
+
+#### Examples
+
+In:
+
+```jsx
+import { Table, TableBody, TableHeader, TableProps } from "@patternfly/react-table";
+```
+
+Out:
+
+```jsx
+import {
+ Table as TableDeprecated,
+ TableBody as TableBodyDeprecated,
+ TableHeader as TableHeaderDeprecated,
+ TableProps as TablePropsDeprecated
+} from '@patternfly/react-table/deprecated';```
```

Comment:
I think if the backticks aren't on a newline they'll cause some formatting issues.